honeybot/plugins/quote.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""
+[quote.py]
+Quotes Plugin
+
+[Author]
+German Corpaz
+
+[About]
+Sends a random quote on request
+
+[Commands]
+>>> .quote
+returns a random quote
+"""
+import random
+
+
+class Plugin:
+    def __init__(self):
+        pass
+
+    def quote(self):
```

Comment:
adding quotes author name would have been nice. btw just added more plugins dev info in readme